### ISSUE TITLE
5.1.0 error.team_not_found message when trying to change channel

### ISSUE THREAD

**Issue by komodin** [mattermost/desktop#2133]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: https://docs.mattermost.com/install/troubleshooting.html](https://docs.mattermost.com/install/troubleshooting.html).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
After upgrading desktop app to 5.1 on Windows 10, some users sees the error.team_not_found message: "The team you're requesting is private or does not exist. Please contact your Administrator for an invitation.".

When looking at the developer tools, we see a 404 on requests to `/api/v4/teams/name/messages`.

**Environment**
- Operating System:
- Mattermost Desktop App version: 5.1.0
- Mattermost Server version: 6.3.8 Enterprise Edition (Starter)

**Steps to reproduce**
Open MatterMost desktop client (it drops you to General channel) and then click on another channel.

**Expected behavior**
Open the other channel

**Observed behavior**

![Imagen Pegada el 2022-5-26 12-21](https://user-images.githubusercontent.com/383352/170534393-8bd7b77c-1331-4781-98d5-739d8f5d865a.png)

![Imagen Pegada el 2022-5-26 12-29](https://user-images.githubusercontent.com/383352/170527028-477aa906-74da-4906-a468-df8106eec4c8.png)

**Comment by devinbinnie** [comment]
Can you confirm that the URL you have configured in the Desktop App matches the SiteURL configured on your server?
If so, can you turn on Debug logging in the Settings window and send the logs to me? If you need to keep things private, feel free to DM me at `@devin.binnie` on `community.mattermost.com` 

Thanks!

**Comment by komodin** [comment]
Hi @devinbinnie ! Thanks for the quick answer.
 I think this may be related to #2125.
We've found that the configured URL in the desktop app for the users that have the error includes the teamname, like `https://mm.ourcompany.com/ourteam`, instead of just `https://mm.ourcompany.com`.

I've also tried it on my client (macOS) and I can replicate the issue.

Let me know if you still need the debug logs and I'll reach you there.

Thanks!
